config/routes.js

```diff
@@ -17,6 +17,11 @@ module.exports = function (app, passport, auth) { // eslint-disable-line
   app.get('/chooseavatars', users.checkAvatar);
   app.get('/signout', users.signout);
 
+  app.post('/api/selected-region', (req) => {
```

Comment:
we can emit the region selected when we call start game before loading the questions, this endpoint would not be required